appveyor.yml

```diff
@@ -9,16 +9,22 @@ matrix:
 
 cache:
   - C:\Users\appveyor\.cargo\registry
+  - C:\tools\vcpkg\installed\
   - target
 
 install:
   - appveyor-retry appveyor DownloadFile https://win.rustup.rs/ -FileName rustup-init.exe
   - rustup-init.exe -y --default-toolchain %CHANNEL% --default-host %TARGET%
   - set PATH=%PATH%;C:\Users\appveyor\.cargo\bin
+  - C:\tools\vcpkg\vcpkg.exe integrate install
+  - C:\tools\vcpkg\cmake -DCMAKE_TOOLCHAIN_FILE=c:/tools/vcpkg/scripts/buildsystems/vcpkg.cmake
+  - vcpkg install sqlite3:x86-windows
+  - vcpkg install sqlite3:x64-windows
   - rustc -V
   - cargo -V
 
 build: false
 
 test_script:
   - cargo check
+  - cargo build
```

Comment:
We can probably remove `cargo check` if we add `cargo build`.

Reply:
good catch, when `build` is working we should remove `check`